dependencies/list_versions.py

```diff
@@ -23,244 +38,306 @@
 from os import linesep, path
 from sys import argv, exit, stdout, stderr
 from getopt import getopt, GetoptError
-from subprocess import Popen, PIPE, STDOUT
+from subprocess import Popen, PIPE
 from pkg_resources import parse_version
 from itertools import groupby
 from six.moves import filter
 import shlex
 import re
 
 # See also: /usr/include/sysexits.h
-_ERROR_RUNTIME=1
-_ERROR_ILLEGAL_ARGS=64
-
-_TAG_REGEX='^[0-9a-f]{40}\s+refs/tags/%s$'
-_VERSION='version'
-DEPS={
-	'go': {
-		'url': 'https://github.com/golang/go',
-		're':  'go(?P<%s>[\d\.]+(?:rc\d)*)' % _VERSION,
-		'min': None
-	},
-	'docker': {
-		'url': 'https://github.com/docker/docker',
-		're':  'v(?P<%s>\d+\.\d+\.\d+(?:\-rc\d)*)' % _VERSION,
-		# Weave Net only works with Docker from 1.10.0 onwards, so we ignore all previous versions:
-		'min': '1.10.0'
-	},
-	'kubernetes': {
-		'url': 'https://github.com/kubernetes/kubernetes',
-		're':  'v(?P<%s>\d+\.\d+\.\d+(?:\-beta\.\d)*)' % _VERSION,
-		# Weave Kube requires Kubernetes 1.4.2+, so we ignore all previous versions:
-		'min': '1.4.2'
-	}
+_ERROR_RUNTIME = 1
+_ERROR_ILLEGAL_ARGS = 64
+
+_TAG_REGEX = '^[0-9a-f]{40}\s+refs/tags/%s$'
+_VERSION = 'version'
+DEPS = {
+    'go': {
+        'url': 'https://github.com/golang/go',
+        're': 'go(?P<%s>[\d\.]+(?:rc\d)*)' % _VERSION,
+        'min': None
+    },
+    'docker': {
+        'url': 'https://github.com/docker/docker',
+        're': 'v(?P<%s>\d+\.\d+\.\d+(?:\-rc\d)*)' % _VERSION,
+        # Weave Net only works with Docker from 1.10.0 onwards, so we ignore
+        # all previous versions:
+        'min': '1.10.0',
+    },
+    'kubernetes': {
+        'url': 'https://github.com/kubernetes/kubernetes',
+        're': 'v(?P<%s>\d+\.\d+\.\d+(?:\-beta\.\d)*)' % _VERSION,
+        # Weave Kube requires Kubernetes 1.4.2+, so we ignore all previous
+        # versions:
+        'min': '1.4.2',
+    }
 }
 
+
 class Version(object):
-	''' Helper class to parse and manipulate (sort, filter, group) software versions. '''
-	def __init__(self, version):
-		self.version = version
-		self.digits = [int(x) if x else 0 for x in re.match('(\d*)\.?(\d*)\.?(\d*).*?', version).groups()]
-		self.major, self.minor, self.patch = self.digits
-		self.__parsed = parse_version(version)
-		self.is_rc = self.__parsed.is_prerelease
-	def __lt__ (self, other):
-		return self.__parsed.__lt__(other.__parsed)
-	def __gt__ (self, other):
-		return self.__parsed.__gt__(other.__parsed)
-	def __le__ (self, other):
-		return self.__parsed.__le__(other.__parsed)
-	def __ge__ (self, other):
-		return self.__parsed.__ge__(other.__parsed)
-	def __eq__ (self, other):
-		return self.__parsed.__eq__(other.__parsed)
-	def __ne__ (self, other):
-		return self.__parsed.__ne__(other.__parsed)
-	def __str__(self):
-		return self.version
-	def __repr__(self):
-		return self.version
+    ''' Helper class to parse and manipulate (sort, filter, group) software
+    versions. '''
```

Comment:
The wrapping was me rather than yapf, I expect it's a missing feature rather than a deliberate policy.
flake8 complains appropriately.